pg_duckdb-motherduck/benchmark.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+set -e
+
+#sudo apt-get update
```

Comment:
Could we throw l. 5-12 out?

Reply:
removed the dual sets of installation instructions